### PR TITLE
Add exposureTime property

### PR DESCRIPTION
**Pull request by bringert**
See https://github.com/w3c/mediacapture-image/issues/199

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -690,6 +706,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><i><dfn>Exposure Compensation</dfn></i> is a numeric camera setting that adjusts the exposure level from the current value used by the implementation.  This value can be used to bias the exposure level enabled by auto-exposure, and usually is a symmetric range around 0 EV (the no-compensation value).</li>
 
+    <li><i><dfn>Exposure Time</dfn></i> is a numeric camera setting that controls the length of time during which light is allowed to fall on the photosensitive device. This value is used in manual exposure mode to control exposure. The value is in 100 microsecond units. That is, a value of 1.0 means an exposure time of 1/10000th of a second and a value of 10000.0 means an exposure time of 1 second.</li>
```
Comment (yellowdoge) [inline index.bs]: Please s/manual exposure mode/`{{MeteringMode/manual}} {{exposureMode}}`/ for better autolinking.
Same in l.699.

File: index.bs [hunk 2]
```diff
@@ -690,6 +706,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><i><dfn>Exposure Compensation</dfn></i> is a numeric camera setting that adjusts the exposure level from the current value used by the implementation.  This value can be used to bias the exposure level enabled by auto-exposure, and usually is a symmetric range around 0 EV (the no-compensation value).</li>
```
Comment (yellowdoge) [inline index.bs]: This is only applicable if `exposureMode` is `continuous` or `single-shot`, right?

File: index.bs [hunk 3]
```diff
@@ -606,6 +619,9 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
   <dt><dfn dict-member for="MediaTrackSettings"><code>exposureCompensation</code></dfn></dt>
   <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation.</dd>
 
+  <dt><dfn dict-member for="MediaTrackSettings"><code>exposureTime</code></dfn></dt>
+  <dd>Current <a>exposure time</a> setting. This field is only significant if {{MediaTrackSettings/exposureMode}} is {{manual}}.</dd>
```
Comment (yellowdoge) [inline index.bs]: Correspondingly, l.620 should say it's only significant if `{{MediaTrackSettings/exposureMode}}` is not `{{manual}}` ?

File: index.bs [hunk 4]
```diff
@@ -604,7 +617,10 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
   <dd><a>Color temperature</a> in use for the white balance calculation of the scene. This field is only significant if {{MediaTrackSettings/whiteBalanceMode}} is {{manual}}.</dd>
 
   <dt><dfn dict-member for="MediaTrackSettings"><code>exposureCompensation</code></dfn></dt>
-  <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation.</dd>
+  <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation. This field is only significat if {{MediaTrackSettings/exposureMode}} is </dd>
```
Comment (yellowdoge) [inline index.bs]: s/`significat if {{MediaTrackSettings/exposureMode}} is`/`significant if {{MediaTrackSettings/exposureMode}} is {{MeteringMode/continuous}} or {{MeteringMode/single-shot}}`/

File: index.bs [hunk 5]
```diff
@@ -459,6 +464,9 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dt><dfn dict-member for="MediaTrackCapabilities"><code>exposureCompensation</code></dfn></dt>
   <dd>This reflects the supported range of <a>exposure compensation</a>. The supported range can be, and usually is, centered around 0 EV.</dd>
 
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>exposureTime</code></dfn></dt>
+  <dd>This reflects the supported range of <a>exposure time</a>.</dd>
```
Comment (yellowdoge) [inline index.bs]: For completion, could you add here something like `Values are numeric. Increasing values indicate increasing exposure time` to mimic the other entries?

Comment (bringert) [reply index.bs]: Done

**Pull request merged**